src/pages/Checkout/CheckoutPage.vue

```diff
@@ -493,6 +498,21 @@ export default {
 		// TODO: Implement check against contentful setting
 		// to signify if holiday mode is enabled
 
+		// VUE-1725 Async Checkout Mutation Rollout
+		const asyncCheckoutExp = this.apollo.readFragment({
+			id: `Experiment:${ASYNC_CHECKOUT_EXP}`,
+			fragment: experimentVersionFragment,
+		}) || {};
+
+		this.asyncCheckoutExpVersion = asyncCheckoutExp?.version;
+		if (this.asyncCheckoutExpVersion && !this.emptyBasket) {
```

Comment:
Do we only want to store whether the exp is enabled if `!this.emptyBasket`?

Reply:
Correct. no need to track or swap mutations if no checkout can take place. Adding something to the basket would activate this block.

Reply:
In that case, should we move setting the data prop into the if?

Reply:
Yea, I suppose we could move the readFragment and setting the version var into a check for `!emptyBasket`